bin/load-spark-env.cmd

```diff
@@ -29,10 +29,7 @@ if [%SPARK_ENV_LOADED%] == [] (
     set SPARK_CONF_DIR=%~dp0..\conf
   )
 
-  set SPARK_ENV_CMD=%SPARK_CONF_DIR%\%SPARK_ENV_CMD%
```

Comment:
Can't you just replace `%SPARK_ENV_CMD%` to  `spark-env.cmd`?

Reply:
This way also doesn't work.

Reply:
I think you can just remove `SPARK_ENV_CMD` here. Don't bother about the commented codes. I am fixing it now.

Reply:
Last comment. Can you remove SPARK_ENV_CMD on the top? It's not used anymore.

Reply:
Last comment. Can you remove SPARK_ENV_CMD on the top? It's not used anymore